sunpy/map/sources/gong.py

```diff
@@ -61,3 +71,56 @@ def spatial_units(self):
     @property
     def observer_coordinate(self):
         return get_earth(self.date)
+
+
+class GONGHalphaMap(GenericMap):
```

Comment:
So as far I as I know GONG produces high cadence H-alpha, lower cadence ~1 hours magnetograms, and the carrington rotation magnetic maps.

Reply:
I guess using SDO as the example, we have separate map source classes for the different instruments on the same observatory so it would make sense to follow that same logic here, eg have an H-alpha source and a magnetogram source.